AppInspector/Commands/AnalyzeCommand.cs

```diff
@@ -641,27 +654,51 @@ private void UnZipAndProcess(string filename, ArchiveFileType archiveFileType, b
 
             try
             {
-                IEnumerable<FileEntry> files = Extractor.ExtractFile(filename);
+                IEnumerable<FileEntry> files = Extractor.ExtractFile(filename).Where(x => x != null);
 
-                if (files.Count() > 0)
+                if (_options.SingleThread)
                 {
-                    _metaDataHelper.Metadata.TotalFiles += files.Count();//additive in case additional child zip files processed
-
                     foreach (FileEntry file in files)
                     {
-                        //check uncompressed file passes standard checks
-                        LanguageInfo languageInfo = new LanguageInfo();
-                        if (FileChecksPassed(file.FullPath, ref languageInfo, file.Content.Length))
+                        try
+                        {
+                            //check uncompressed file passes standard checks
+                            LanguageInfo languageInfo = new LanguageInfo();
+                            if (FileChecksPassed(file.FullPath, ref languageInfo, file.Content.Length))
+                            {
+                                byte[] streamByteArray = file.Content.ToArray();
+                                ProcessInMemory(file.FullPath, Encoding.UTF8.GetString(streamByteArray, 0, streamByteArray.Length), languageInfo);
+                            }
+                        }
+                        catch (Exception)
                         {
-                            byte[] streamByteArray = file.Content.ToArray();
-                            ProcessInMemory(file.FullPath, Encoding.UTF8.GetString(streamByteArray, 0, streamByteArray.Length), languageInfo);
+                            Console.WriteLine($"Failed to parse {file.FullPath}");
                         }
```

Comment:
Change msg to Failed to decompress file {file.FullPath} not parse for user clarity and change Console.WriteLine WriteOnce.SafeLog.  I think since we're handling parrellel processing now the final output should include a msg to check the logs for any notices and/or track if there were non-fatal exceptions written to the log and let them know.

Reply:
Fixed in 560b8bf.